html/links/icon/no-error-event.sub.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>&lt;link rel="icon"&gt; doesn't fire an error event</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+  async_test((t) => {
+    const link = document.createElement("link");
+    link.rel = "icon";
+    link.href =
+      "http://not.{{domains[]}}:{{ports[http][0]}}/images/wpt-logo/wpt-logo-darkblue-bg.svg";
```

Comment:
Is the intention just to get a 404? Why not use a simpler URL, like `/404`?

Or if you're trying to get a network error, then use `{{hosts[][nonexistent]}}`.

Reply:
The intention was to get a network error, but I didn't know `{{hosts[][nonexistent]}}` was a thing.